frontend/src/index.js

```diff
@@ -159,6 +159,7 @@ async function showFile(source, file, revision, selectedLine) {
     language,
     lines: source.map((line, nb) => {
       const coverage = file.coverage[nb];
+      nb = nb + 1;
```

Comment:
Looks good, but please:
- move it after line 185;
- add a comment before it to explain why we need to do it.

Reply:
Made the changes. Please let me know if anything else is required. :)